exercises/practice/word-search/src/main/java/WordSearcher.java

```diff
@@ -1,10 +1,7 @@
-/*
+import java.util.*;
```

Comment:
```suggestion
import java.util.Map;
import java.util.Optional;
```

Avoiding using the * import, as it can be considered bad practice. It's better to be explicit in this case.